shu-etal.ipynb

```diff
@@ -0,0 +1,1516 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
```

Comment:
I'd be great to add a line to explain what this histogram is and to add axes titles.
Also, all the merging could be done in 1 cell.